src/pages/ProductList/ProductList.js

```diff
@@ -1,8 +1,176 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import SearchBox from './components/SearchBox/SearchBox';
+import CardList from './components/Card/CardList';
+import { useNavigate, useLocation } from 'react-router-dom';
```

Comment:
동일한 패키지에서 import하고 있기 때문에 같이 import하시면 됩니다.
```suggestion
import { Link, useNavigate, useLocation } from 'react-router-dom';
import SearchBox from './components/SearchBox/SearchBox';
import CardList from './components/Card/CardList';
```

Reply:
수정하였습니다.